packages/ui-components/src/exports/button.ts

```diff
@@ -0,0 +1,11 @@
+import { Button } from '../components/button';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'hero-button': Button;
+  }
+}
+
+if (!customElements.get('sme-button')) {
+  customElements.define('sme-button', Button);
+}
```

Comment:
We should probably override this function so we don't have to do this every time.

Reply:
Yes, with extensible prefixes 😎

Reply:
Will be fixed in: https://github.com/JortWillemsen/stryker-dashboard-redesign/pull/25